src/quicktype-core/language/JavaScript.ts

```diff
@@ -273,7 +276,17 @@ export class JavaScriptRenderer extends ConvenienceRenderer {
                     this.emitLine("return JSON.stringify(uncast(value, ", typeMap, "), null, 2);");
                 }
             });
-        });
+        };
+
+        switch (this._jsOptions.converters) {
+            case ConvertersOptions.All:
+                this.forEachObject("interposing", converter);
```

Comment:
#naming-is-hard

Is `All` the right name for this? My assumption is this wont include enums?

Reply:
Renamed to `AllObjects`.